Sources/CCryptoBoringSSLShims/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# This source file is part of the SwiftCrypto open source project
+#
+# Copyright (c) 2021 Apple Inc. and the SwiftCrypto project authors
+# Licensed under Apache License v2.0
+#
+# See LICENSE.txt for license information
+# See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+add_library(CCryptoBoringSSLShims STATIC
+  shims.c)
+
+target_include_directories(CCryptoBoringSSLShims PUBLIC
+  include)
+
+target_link_libraries(CCryptoBoringSSLShims PUBLIC
+  CCryptoBoringSSL)
+  
+set_target_properties(CCryptoBoringSSLShims PROPERTIES
+  INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_Swift_MODULE_DIRECTORY})
+  
+_install_target(CCryptoBoringSSLShims)
```

Comment:
Is there a reason to install the shims?  It is a static library, it does not need to be distributed correct?